compiler/erg_compiler/context/initialize/classes.rs

```diff
@@ -1148,227 +1198,221 @@ impl Context {
+        self.register_builtin_type(arr_t, array_, vis, Const, Some(FUNC_LIST));
+        self.register_builtin_type(set_t, set_, vis, Const, Some(FUNC_SET));
+        self.register_builtin_type(g_dict_t, generic_dict, vis, Const, Some(FUNC_DICT));
+        self.register_builtin_type(dict_t, dict_, vis, Const, Some(FUNC_DICT));
+        self.register_builtin_type(mono(BYTES), bytes, vis, Const, Some(BYTES));
         self.register_builtin_type(
-            mono("GenericTuple"),
+            mono(GENERIC_TUPLE),
             generic_tuple,
             Private,
             Const,
-            Some("tuple"),
+            Some(FUNC_TUPLE),
         );
-        self.register_builtin_type(_tuple_t, tuple_, vis, Const, Some("tuple"));
-        self.register_builtin_type(mono("Record"), record, vis, Const, Some("Record"));
-        self.register_builtin_type(or_t, or, vis, Const, Some("Union"));
+        self.register_builtin_type(_tuple_t, tuple_, vis, Const, Some(FUNC_TUPLE));
+        self.register_builtin_type(mono(RECORD), record, vis, Const, Some(RECORD));
+        self.register_builtin_type(or_t, or, vis, Const, Some(UNION));
         self.register_builtin_type(
-            mono("StrIterator"),
+            mono(STR_ITERATOR),
             str_iterator,
             Private,
             Const,
-            Some("str_iterator"),
+            Some(FUNC_STR_ITERATOR),
         );
         self.register_builtin_type(
-            poly("ArrayIterator", vec![ty_tp(T.clone())]),
+            poly(ARRAY_ITERATOR, vec![ty_tp(T.clone())]),
             array_iterator,
             Private,
             Const,
-            Some("array_iterator"),
+            Some(FUNC_ARRAY_ITERATOR),
         );
         self.register_builtin_type(
-            poly("RangeIterator", vec![ty_tp(T.clone())]),
+            poly(RANGE_ITERATOR, vec![ty_tp(T.clone())]),
             range_iterator,
             Private,
             Const,
-            Some("RangeIterator"),
+            Some(RANGE_ITERATOR),
         );
         self.register_builtin_type(
-            poly("Enumerate", vec![ty_tp(T.clone())]),
+            poly(ENUMERATE, vec![ty_tp(T.clone())]),
             enumerate,
             Private,
             Const,
-            Some("enumerate"),
+            Some(FUNC_ENUMERATE),
         );
         self.register_builtin_type(
-            poly("Filter", vec![ty_tp(T.clone())]),
+            poly(FILTER, vec![ty_tp(T.clone())]),
             filter,
             Private,
             Const,
-            Some("filter"),
+            Some(FUNC_FILTER),
         );
         self.register_builtin_type(
-            poly("Map", vec![ty_tp(T.clone())]),
+            poly(MAP, vec![ty_tp(T.clone())]),
             map,
             Private,
             Const,
-            Some("map"),
+            Some(FUNC_MAP),
         );
         self.register_builtin_type(
-            poly("Revered", vec![ty_tp(T.clone())]),
```

Comment:
Just a quick confirmation, is this a typo of "Reversed"? 🤔

Reply:
Oh, yes. "Reversed" is right.

Reply:
Thanks 😊